apps/gadgetron/Server.cpp

```diff
@@ -1,15 +1,20 @@
 
 #include <boost/asio.hpp>
 
+#include "../clients/gadgetron_ismrmrd_client/gadgetron_ismrmrd_client.cpp"
```

Comment:
Including a cpp file is almost always an indication that you are doing something incorrect.